.github/workflows/standard.yml

```diff
@@ -0,0 +1,24 @@
+name: PCDS Standard Testing
+
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - created
+
+jobs:
+  standard:
+    uses: pcdshub/pcds-ci-helpers/.github/workflows/python-standard.yml@master
+    with:
+      # The workflow needs to know the package name.  This can be determined
+      # automatically if the repository name is the same as the import name.
+      package-name: "typhos"
+      # Extras that will be installed for both conda/pip:
+      testing-extras: "happi"
+      # Extras to be installed only for conda-based testing:
+      conda-testing-extras: "pcdsdevices pyqt=5"
```

Comment:
Let's add pyqt/pyqt5 to the conda-recipe and requirements.txt, respectively
And `pcdsdevices` to the conda recipe